frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -61,6 +61,13 @@ import { SessionRecordingPlayerExplorerProps } from './view-explorer/SessionReco
 export const PLAYBACK_SPEEDS = [0.5, 1, 1.5, 2, 3, 4, 8, 16]
 export const ONE_FRAME_MS = 100 // We don't really have frames but this feels granular enough
 
+export interface PlayerTimeTracking {
+    state: 'buffering' | 'playing' | 'paused' | 'errored' | 'ended' | 'unknown'
```

Comment:
collapses to tracking one state instead of trying to track two different ones... since _really_ you can't be playing and buffering at the same time

(technically we do both, but in terms of time tracking we are either playing, buffering, or not)